pkg/version/version.go

```diff
@@ -0,0 +1,4 @@
+package version
+
+// Version is the version of the build.
+const Version = "0.15.0-dev"
```

Comment:
`0.14.1-dev` to be consistent with the other libraries where we always assume the next release to be a patch one.

Reply:
Oh scratch that!

Reply:
Fixed :+1:

Reply:
0.15.0 is perfect as we need 0.14.1 in the 0.14 branch